Podcast/SearchITunesViewController.swift

```diff
@@ -134,6 +134,10 @@ class SearchITunesViewController: ViewController, UITableViewDelegate, UITableVi
         searchController.searchBar.becomeFirstResponder()
     }
     
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
```

Comment:
We don't want to be fetchingData upon text change. This might overwhelm our servers with unnecessary requests of `G`..`Gu`..`Guy` .. etc. Since iTunes rate limits us we only want to call this endpoint when the user presses search